src/test/javascript/spec/component/exam/manage/exam-management.service.spec.ts

```diff
@@ -32,7 +34,7 @@ describe('Exam Management Service Tests', () => {
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            providers: [ExamManagementService],
+            providers: [ExamManagementService, { provide: AccountService, useClass: MockAccountService }],
```

Comment:
```suggestion
            providers: [ExamManagementService, MockProvider(AccountService)],
```
With the corresponding import. I sadly still did not receive any response on which approach should be prefferred